File: Python/Start/统计好三元组（Easy）.py
class Solution:
    def count_good_triplets(self, arr: List[int], a: int, b: int, c: int) -> int:
        ans = 0
        arr_size = len(arr)
        for i in range(arr_size):
            for j in range(i + 1, arr_size):
                if abs(arr[i] - arr[j]) >a:
                    continue
                for k in range(j + 1, arr_size):
                    if abs(arr[j] - arr[k]) <= b and abs(arr[i] - arr[k]) <= c:
                        ans += 1
        return ans 

File: Cpp/Start/统计好三元组（Easy）.cpp
#include <vector>
#include <cmath>

class Solution 
{
public:
    int count_good_triplets(std::vector<int>& arr, int a, int b, int c) 
    {
        int len = arr.size();
        int ans = 0;
        for (int i = 0; i < len; ++i)
        {
            for (int j = i + 1; j < len; ++j)
            {
                if (abs(arr[i] - arr[j]) > a) continue;
                for (int k = j + 1; k < len; ++k)
                {
                    if (abs(arr[j] - arr[k]) <= b && abs(arr[i] - arr[k]) <= c) ans++;
                }
            }
        }
        return ans;
    }
};